src/index.html

```diff
@@ -3,11 +3,379 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBiKE</title>
```

Comment:
horizontal scroll appears on width < 336px
however when viewing in responsive dimension it seems fine

Reply:
I can't see the horizontal scrolling